### ISSUE TITLE
Change color workbench.action.openLargerOutput in jupyter notebook

### ISSUE THREAD

**Issue by vuthanhdatt** [Binaryify/OneDark-Pro#612]
I'm trying to change color of command open larger output in jupyter notebook but didn't success. I'm also look through all annotation in vscode color reference but not finding what i'm looking for. Anyone can help?
![image](https://user-images.githubusercontent.com/80867452/134685489-6b9ecf1c-6907-4427-9131-cd80920b6d2c.png)

**Comment by Binaryify** [comment]
could you provide some code to reproduce

**Comment by vuthanhdatt** [comment]
@Binaryify I'm printing geojson data of this code 

```
import plotly.express as px
df = px.data.election()
geojson = px.data.election_geojson()
```

**Comment by vuthanhdatt** [comment]
I'm currently using One Dark Pro default and don't have any overwrite color customize

**Comment by Binaryify** [comment]
@vuthanhdatt could you tell me how to run that code and print the result in your screenshot

**Comment by Binaryify** [comment]
oh, I know how to print that now

**Comment by Binaryify** [comment]
I might need create a issue in VSCODE's repo, bucause vscode didn't provide setting about it

**Comment by Binaryify** [comment]
you can follow this issue
https://github.com/microsoft/vscode/issues/133739

**Comment by vuthanhdatt** [comment]
ok, thanks

**Comment by Binaryify** [comment]
this problem had been fixed in vscode v1.62.0
